packages/calling-stateful-client/src/modifiers/ClearErrors.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CallClientState, CallErrorTarget } from '../CallClientState';
+import { CallStateModifier } from '../StatefulCallClient';
+
+/**
+ * Creates a new {@link CallStateModifier} that clears the latest error for specific targets.
+ *
+ * Intended to be used with {@link StatefulCallClient.modifyState}.
+ *
+ * @param targets CallErrorTarget[] to clear the errors for.
+ */
+export const newClearCallErrorsModifier = (targets: CallErrorTarget[]): CallStateModifier => {
```

Comment:
why is it starting with new? Is it like something "createNewClearCallErrorModifier"?

Reply:
Naming convetion `newX` is quite common for factory functions (the words `create` and `new` are redundant. either `createX` or `newX` is sufficient. `newX` is shorter, so I prefer that).

This naming was discussed in the chat modifiers PR as well :)